### PR TITLE
fix #1802 and #1803

### LINKED ISSUE #1802: Cannot add JWT auth plugin "zExceptions.BadRequest: id_"
Vanilla plone5.2

- Open ZMI/acl_users
- choose to add JWT auth plugin
- submit the add form

```
Traceback (innermost last):
  Module ZPublisher.WSGIPublisher, line 176, in transaction_pubevents
  Module ZPublisher.WSGIPublisher, line 385, in publish_module
  Module ZPublisher.WSGIPublisher, line 280, in publish
  Module ZPublisher.mapply, line 78, in mapply
  Module ZPublisher.WSGIPublisher, line 73, in missing_name
  Module ZPublisher.HTTPResponse, line 1029, in badRequestError
zExceptions.BadRequest: id_
```
Found that the input field name attribute should be "id_" 
here -> https://github.com/plone/plone.restapi/blob/37d02e7772b1b402ef74b8e55bc7ea4ad786d014/src/plone/restapi/pas/add_plugin.zpt#L17

Should be:  ` <input id="id_" name="id_" class="form-control" type="text" tal:attributes="value request/id_|string:jwt_auth" />`

alessandro.

### PR DESCRIPTION
**Pull request by sauzher**
----
📚 Documentation preview 📚: https://plonerestapi--1804.org.readthedocs.build/

### CODE REVIEW

File: news/1802.bugfix [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Fixes adding JWT plugin
```
Comment (stevepiercy) [inline news/1802.bugfix:1]: This is more explicit, and gives you credit for the fix.

```suggestion
Fixed implementation of the `jwt_auth` plugin. It now can be added, and its properties, `use_keyring` and `store_tokens`, can be updated. @sauzher
```

File: news/1802.bugfix [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+Fixes changing plugin properties
```
Comment (stevepiercy) [inline news/1802.bugfix:2]: Merged with above.

```suggestion
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
I only reviewed the news item. This needs a technical review.

**Review by stevepiercy — APPROVED** [review]
News looks good. Needs a technical review.

**Review by davisagli — APPROVED** [review]
@sauzher Thanks!

**Comment by davisagli** [comment]
@jenkins-plone-org please run jobs

**Pull request merged**